### ISSUE TITLE
InvalidXmlRequest: The request body's XML was invalid or not correctly specified.

### ISSUE THREAD

**Issue by dkarzon** [JetBrains/teamcity-azure-plugin#19]
I have recently updated my Teamcity instance to v9.1.3 (build 37176) and the azure plugin to v0.2.7 and the plugin has stopped working with the below exception happening when attempting to start or stop one of my VMs.

```
com.microsoft.windowsazure.exception.ServiceException: InvalidXmlRequest: The request body's XML was invalid or not correctly specified.
	at com.microsoft.windowsazure.exception.ServiceException.createFromXml(ServiceException.java:212)
	at com.microsoft.windowsazure.management.compute.VirtualMachineOperationsImpl.beginShutdown(VirtualMachineOperationsImpl.java:3619)
	at jetbrains.buildServer.clouds.azure.connector.AzureApiConnector.stopVM(AzureApiConnector.java:394)
	at jetbrains.buildServer.clouds.azure.AzureCloudImage$1.action(AzureCloudImage.java:136)
	at jetbrains.buildServer.clouds.azure.connector.ProvisionActionsQueue$1.execute(ProvisionActionsQueue.java:74)
	at jetbrains.buildServer.clouds.azure.connector.ConditionalRunner.run(ConditionalRunner.java:43)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.runAndReset(FutureTask.java:308)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(ScheduledThreadPoolExecutor.java:180)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:294)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
```

**Comment by paksv** [comment]
There's no version 0.2.7. Can you please clarify what is your plugin version?

**Comment by dkarzon** [comment]
Strange, that's what it says in my TeamCity. 
![image](https://cloud.githubusercontent.com/assets/214449/11257260/51b0f2a8-8eb5-11e5-80bb-873edde79d8e.png)

Though I'm pretty sure it was `v0.2.63` downloaded from your teamcity instance.

**Comment by paksv** [comment]
can you please try to update the plugin to 0.2.63?

**Comment by paksv** [comment]
Does the problem occurs, when you open the cloud profile? Does it populate values properly?

**Comment by dkarzon** [comment]
Yep it's able to seem my VMs in Azure just that error happens when attempting to start/stop them.

**Comment by paksv** [comment]
Did you create this VM in the new portal version (Virtual machines tab)? Current plugin version is incompatible with Azure Resource Manager Templates (#11)

**Comment by sdurandeu** [comment]
This is also happening on 0.2.62, when the plugin tries to stop the VM.

**Comment by dkarzon** [comment]
No this was an old VM created a while ago. I think an older version of the plugin was working with it before (`v0.2.55` I believe)

**Comment by paksv** [comment]
The bug arrived along with Azure SDK v0.8.0. I fixed the issue by updating to 0.9.0. 

Please check if #0.2.67 helps at https://teamcity.jetbrains.com/viewType.html?buildTypeId=TeamcityAzurePlugin_Build

**Comment by dkarzon** [comment]
@paksv Just updated and can confirm this is fixed! Thanks for the quick response :+1:
